activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -45,130 +45,780 @@ def primary_key(name, type = :primary_key, **options)
+        #
+        # To rename an enum value:
+        #
+        #   def change
+        #     rename_enum_value :article_state, from: "archived", to: "deleted"
+        #   end
+        #
+        # To inspect all enum types and values in the database you can use this query in +bin/rails db+ or +psql+ console:
+        #
+        #   SELECT n.nspname AS enum_schema,
+        #          t.typname AS enum_name,
+        #          e.enumlabel AS enum_value
+        #     FROM pg_type t
+        #          JOIN pg_enum e ON t.oid = e.enumtypid
+        #          JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace;
 
         ##
         # :method: hstore
         # :call-seq: hstore(*names, **options)
+        #
+        # Adds an +hstore+ column for storing sets of key-value pairs.
+        #
+        # PostgreSQL’s +hstore+ type allows you to store a dynamic set of string keys and values
+        # in a single column. This is useful for flexible or semi-structured data such as user
+        # preferences, metadata, or settings.
+        #
+        # You must enable the +hstore+ extension to use this type:
+        #
+        #   enable_extension "hstore" unless extension_enabled?("hstore")
+        #
+        # Example:
+        #
+        #   # db/migrate/20131009135255_create_profiles.rb
+        #   class CreateProfiles < ActiveRecord::Migration[8.1]
+        #     def change
+        #       enable_extension "hstore" unless extension_enabled?("hstore")
+        #
+        #       create_table :profiles do |t|
+        #         t.hstore :settings
+        #       end
+        #     end
+        #   end
+        #
+        #   # app/models/profile.rb
+        #   class Profile < ApplicationRecord
+        #   end
+        #
+        #   Profile.create(settings: { "color" => "blue", "resolution" => "800x600" })
+        #
+        #   irb> profile = Profile.first
+        #   irb> profile.settings
+        #   # => {"color"=>"blue", "resolution"=>"800x600"}
+        #
+        #   irb> profile.settings = { "color" => "yellow", "resolution" => "1280x1024" }
+        #   irb> profile.save!
+        #
+        #   irb> Profile.where("settings -> 'color' = ?", "yellow")
+        #
+        # 🔗 See also: {PostgreSQL type definition}[https://www.postgresql.org/docs/current/static/hstore.html] and {Hstore functions and operators}[https://www.postgresql.org/docs/current/static/hstore.html#id-1.11.7.26.5]
+        #
+        # 💡 When to use +hstore+ vs +jsonb+?
+        #
+        # Use +hstore+ when you need a simple, flat key-value store where both keys and values are strings.
+        # It’s lightweight and efficient for storing unstructured settings or metadata. If you require nested
+        # structures, typed values (e.g. numbers, booleans), or advanced querying capabilities, consider using
+        # +jsonb+ instead.
```

Comment:
Avoiding "you", maybe something like:
```suggestion
        # Use +hstore+ for a simple, flat key-value store where both keys and values are strings.
        # It’s lightweight and efficient for storing unstructured settings or metadata. If nested structures are
        # required, typed values (e.g. numbers, booleans), or advanced querying capabilities, consider using
        # +jsonb+ instead.
```